cmd/gendeepcopy/deep_copy.go

```diff
@@ -79,7 +82,11 @@ func main() {
 		glog.Fatalf("error writing package line: %v", err)
 	}
 
-	versionPath := path.Join(pkgBase, group, version)
+	versionPath := path.Join(pkgBase, *groupVersion)
+	if group == "experimental" {
+		//TODO: we should rename the direcotry /expapi to /experimental, so directory path match groupVerson
```

Comment:
Actually in the TODO list I'm planning moving it to pkg/experiemental. pkg/api/experimental makes more sense.